apps/teca_deeplabv3p_ar_detect.in

```diff
@@ -35,6 +35,11 @@ def main():
         help='name of variable with integrated vapor transport (IVT)'
     )
 
+    parser.add_argument(
+        '--num_threads', type=int, default=-1,
+        help='number of threads used in torch for parallelizing CPU operations'
+    )
```

Comment:
Would you mind reformatting this? Here's what I think we came to in terms of Python style: function arguments should be on the same line as the function call or definition. Wrap function arguments to a new line at 80 chars if they would extend beyond 80. wrapped lines would be indented to the opening paren, unless that would extend beyond 80 chars in which case wrapped lines would be indented just one level. Closing paren directly following the last argument on the same line.

in this case it would result in something like this:
```
    parser.add_argument( '--num_threads', type=int, default=-1,
        help='number of threads used in torch for parallelizing CPU operations')
```

Reply:
Done!